app/cdap/components/DataPrep/store/index.ts

```diff
@@ -89,6 +89,27 @@ export interface IDataPrepState {
   targetDataModel?: IDataModel;
   targetModel?: IModel;
   connectorsWithIcons?: any;
+  recipe: IRecipeData;
+}
+
+export interface IRecipeData {
```

Comment:
plz remove them from here and import from RecipeForm/types
plz do not keep redundant code

this is wrong practise to keep in multiple places

Reply:
removed redundant code